### PR TITLE
Apply masking for control plane logs

### PR DESCRIPTION
**Pull request by AmartC**
This PR introduces masking for control plane logs specifically for InfoS log formats and also will mask Go file paths. For example, a message in this format:

I1129 22:26:22.800223       1 deployment/deployment_controller.go:578] "Finished syncing deployment" deployment="opni/opni-svc-metrics" duration="482.277µs"

This will now be masked as:
```
<klog_date> <num> <go_file_path> finished syncing deployment deployment = <deployment> duration = <duration>
```

This PR also removes writing some of the prior Opensearch fields that were related to workload logs as those are currently no longer in use.

### CODE REVIEW

File: preprocessing-service/preprocess.py [hunk 1]
```diff
@@ -175,18 +173,24 @@ async def mask_logs(queue):
             payload_data_df["kubernetes_component"] = payload_data_df[
                 "kubernetes.labels.component"
             ]
+        masked_logs = []
```
Comment (tybalex) [inline preprocessing-service/preprocess.py:176]: why moving those 4 lines from 115 to here?

File: preprocessing-service/preprocess.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ async def doc_generator(df):
     df["anomaly_predicted_count"] = 0
     df["nulog_anomaly"] = False
     df["drain_anomaly"] = False
+    df["drain_control_plane_template_matched"] = ""
```
Comment (tybalex) [inline preprocessing-service/preprocess.py:55]: does this ever being used somewhere?

Comment (AmartC) [reply preprocessing-service/preprocess.py:55]: Yes, it's used in the DRAIN service.

File: preprocessing-service/preprocess.py (hunk 1 above)
Comment (AmartC) [reply preprocessing-service/preprocess.py:176]: That way, I can apply the masking after it is determined which log is a control plane log.

**Review by tybalex — APPROVED** [review]
lgtm

**Review by tybalex — APPROVED** [review]

**Pull request merged**